docs/arcaflow/workflows/flow-control.md

```diff
@@ -107,3 +107,140 @@ outputs:
     my_plugin: !expr $.steps.my_plugin.outputs.success
     my_other_plugin: !expr $.steps.my_other_plugin.outputs.success
 ```
+
+### Reduce Repetition with `bindConstants()`
+
+The builtin function [bindConstants()](expressions.md#functions) allows you to 
+avoid repeating input variables for a foreach subworkflow. In the example 
+below, the input variable `name`'s value is repeated across each iteration in 
+this input. This requires a more repetitive input and schema definition. This 
+section will show you how to simplify it. 
+
+```yaml title="input-repeat-name.yaml"
```

Comment:
You've still got the H3 header above introducing the idea of `bindConstants` and the H4 header below for the "reduced" version, but it'd be better to have another H4 *here* setting off the "unreduced/original/old" example from the introductory paragraph above.

Reply:
i moved the header